src/workloads/execution/BackgroundIndexConstruction.yml

```diff
@@ -0,0 +1,156 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: |
+  This workload tests the write performance impact of creating indexes on a very large collection.
+  It synthetically lowers the number of concurrent operations allowed to represent a saturated
+  server.
+
+Keywords:
+- stress
+- indexes
+- InsertRemove
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 512
+      socketTimeoutMS: 3600000  # = 1 hour
+
+Actors:
+# Phase 1: Insert enough data to ensure than an index build on each field spills to disk with a
+# memory limit of 100MB.
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 10000000
```

Comment:
Q: Can we also use the thousands separators here to make it more readable?

Reply:
Done